docker-compose-addons.yml

```diff
@@ -223,9 +89,5 @@ services:
         max-size: '10m'
         max-file: '5'
 
-volumes:
-  pgdb:
-    driver: local
-
 networks:
```

Comment:
I'm running into the following error:

> imported compose file [/home/wyocvadmin/cvmanager/jpo-cvmanager/docker-compose.yml] defines conflicting network internal

I think this the 'internal' network definition needs to be removed from this file.

Reply:
I removed the internal network for both files. I don't think it is necessary.